shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/screens/SessionScreen.kt

```diff
@@ -347,9 +347,13 @@ private fun FeedbackPanel(
         }
 
         AnimatedVisibility(visible = feedbackExpanded) {
+            var focusRequested by rememberSaveable { mutableStateOf(false) }
```

Comment:
I would keep the state of request inside focusRequester

Reply:
`FocusRequester` is a framework class, so it won't track this sort of thing for us